### PR TITLE
Firebase Functions implementation

### PR DESCRIPTION
**Pull request by awazgyawali**

### CODE REVIEW

**Comment by awazgyawali** [comment]
Till now, functions for notes have been added.

File: src/firebase/notes.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { fireStoreRef } from '.';
+import firebase from 'react-native-firebase';
+
+export async function addNoteForTalk(talkId, note = 'hello world') {
+  var user = firebase.auth().currentUser;
```
Comment (roshangm1) [inline src/firebase/notes.js]: use let instead of var.

Comment (roshangm1) [inline src/firebase/notes.js]: auth is already exported in index.js inside firebase folder, so we can import auth from there.

**Review by roshangm1 — CHANGES_REQUESTED** [review]

File: src/firebase/notes.js (hunk 1 above)
Comment (awazgyawali) [reply src/firebase/notes.js]: Done

Comment (awazgyawali) [reply src/firebase/notes.js]: Used auth from exported function.

**Comment by awazgyawali** [comment]
Added function to add schedule.

**Pull request merged**